### ISSUE TITLE
Add `.gist code { word-wrap: break-word; }` to css before converting to .pdf

### ISSUE THREAD

**Issue by phseiff** [phseiff/github-flavored-markdown-to-html#29]
I should implement exactly what the issue description says, so generated PDF files contain the whole code rather than a dysfunctional scroll bar.

**Comment by phseiff** [comment]
I just tested, and this doesn't seem to affect `code pre`-spans in any way. :/

**Comment by phseiff** [comment]
I think this is a `wontfix` for now, unless someone comes up with a solution for this, so don't expect something anytime soon.
I will leave this issue open for now, though, in case someone has a solution or one comes to my mind all of a sudden.

**Comment by phseiff** [comment]
I tested again, and this actually works:

```css
.gist pre {
 white-space: pre-wrap;       /* css-3 */
 white-space: -moz-pre-wrap;  /* Mozilla, since 1999 */
 white-space: -pre-wrap;      /* Opera 4-6 */
 white-space: -o-pre-wrap;    /* Opera 7 */
 word-wrap: break-word;       /* Internet Explorer 5.5+ */
}
```

This means that
* I should add this to the CSS before converting to PDF
* I should maybe add an extra option (but only to the Python interface) to also use this for html-conversion (?)

**Comment by phseiff** [comment]
Fixed in `v1.13.0` (you can update to this version with `pip3 install --upgrade gh-md-to-html>=1.13.0`).
